.github/workflows/ci.yml

```diff
@@ -366,7 +368,7 @@ jobs:
       ${{needs.build-info.outputs.build-job-description}} PROD images
       ${{needs.build-info.outputs.all-python-versions-list-as-string}}
     runs-on: "${{needs.build-info.outputs.runs-on}}"
-    needs: [build-info, build-ci-images]
+    needs: [build-info, build-ci-images, tests-postgres]
```

Comment:
I suggest to introduce an empty "smoke-tests" job that will be dependent o tests-postgres and use dependency like.

```
[build-info, smoke-tests]
```

This will be much more "generic". Unlike build-info (which is used in "needs.build-info", we do not need `build-ci-images" if "smoke-tests" (or "tests-postgres" depend on it already. Unfortunately github actions dependencies are not transitive and if we need "build-info" outputs, in `needs` clause we need to add it here.

Reply:
i agree, if we want to do something like that... this would be the way to go.